modules/op-pipelines-concepts.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * openshift_pipelines/understanding-openshift-pipelines.adoc
+
+[id='op-pipelines-concepts_{context}']
+= {op-title} concepts
+
+{op-title} provide the following set of standard CRDs that act as the building blocks from which you can assemble a CI/CD pipeline for your application:
+
```

Comment:
I know we have expanded CRD once in the assembly, but since we aim to enable each module to stand on its own, it probably makes sense to expand it the first time we use it in the module here. WDYT?